GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Communication.cfg

```diff
@@ -69,9 +82,22 @@
 
     @mass = 0.003
     @crashTolerance = 8
+    %breakingForce = 250
+    %breakingTorque = 250
     %fuelCrossFeed = False
-    @maxTemp = 673.15
-    %skinMaxTemp = 673.15
+    @maxTemp = 473.15
+    %skinMaxTemp = 473.15
+
+    @MODULE[ModuleDataTransmitter]
+    {
+        @antennaType = DIRECT
+        @antennaCombinable = True
+        %antennaCombinableExponent = 1
+        @antennaPower = 32
```

Comment:
This is what think needs a value ~4-8Mm to give lunar rating without RT installed.  More if it is supposed to be lunar rated with L1 DSN.